changelog_unreleased/mdx/11563.md

```diff
@@ -0,0 +1,18 @@
+#### Add support for MDX 2 comments (#11563 by @wooorm)
+
+This adds support for MDX 2 comments (which are the same as other comments
+inside JSX) in addition to existing MDX 1 comments (HTML-like constructs).
```

Comment:
```suggestion
This adds basic support for MDX 2 comments (which are the same as other comments
inside JSX) in addition to existing MDX 1 comments (HTML-like constructs), only single star is supported right now.
```

How about notice the limitation here?

Reply:
I’m definitely :+1: on `basic`.

Not sure about `, only single star is supported right now.`.
Because there are many limitations to expressions/comments, not just single-line.
And I’m not sure what “single-star” means? What is “single star” about `{/* asd */}`?

Reply:
I saw `Stars are not yet supported` at https://github.com/prettier/prettier/pull/11563#discussion_r713963012 so I was thinking `single star` is the current supported. I'm not a native English speaker, so maybe you explain it in a better way. Anyway, my point is that a simple description about the limitation should be noticed here.

Reply:
Ah, my bad, that was confusingly worded, sorry!

Thanks Joun. I wrote some more about it. I think this is good?